community/contributing/back-end-code.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: single
+title:  "Code Conventions for Server-side Components"
+sidebar:
+  nav: community
+redirect_from: /docs/back-end-code
+---
+
+{% include toc %}
+
+# Code Conventions for Server-side Components
+
+## Choice of Language
+
+### Groovy
```

Comment:
Since this is what we *don't* recommend, perhaps place this at the bottom of the list of languages?